arch/arm/core/Kconfig

```diff
@@ -22,6 +22,7 @@ config CPU_CORTEX_M
 	select ARCH_HAS_TRUSTED_EXECUTION if ARM_TRUSTZONE_M
 	select ARCH_HAS_STACK_PROTECTION if ARM_MPU || CPU_CORTEX_M_HAS_SPLIM
 	select ARCH_HAS_USERSPACE if ARM_MPU
+	select ARCH_HAS_NOCACHE_MEMORY_SUPPORT if ARM_MPU && CPU_CORTEX_M7
```

Comment:
One more thing. ARM_MPU covers also the NXP-variant. But no implementation is provided for nxp-mpu.

Reply:
Good point. I do not have a way to test this on an NPX MPU, so I guess for now I'll just limit it to the ARM MPU.

Reply:
Then I suppose you should fix the Kconfig dependency, too reflect this.